Gems/WarehouseAssets/README.md

```diff
@@ -82,15 +82,37 @@ The `Warehouse_structural` folder contains prefabs comprising the warehouse buil
 
 The folder contains also a `Components` subfolder, containing assets used in these 3 prefabs.
 
-#### Warehouse scene
+### Warehouse scene
 
 ![](docs/images/WarehouseScene.png)
 
-The `WarehouseScene.prefab` contains a sample, ready-to-use warehouse scene, built from prefabs listed in this document.
+The `WarehouseScene.prefab` contains a sample, ready-to-use warehouse scene, built from prefabs listed in this document. It is built as a tree of assets and prefabs defining each element of the warehouse. The top-level objects are shown on the following image:
 
-### Tips on using Warehouse Assets Gem
+![](docs/images/WarehouseSceneStructure.png)
 
-#### Creating Warehouse scene
+- `Global_illumination` contains lights imitating global illumination and `Reflection Probe` objects.
+- `Storage_markings` contains number boards on storage racks.
+- `Floor_markings` contains decal objects with floor lines.
+- `Movables` contains `Storage_on_wheels` prefabs.
+- `Floor_tire_traces` contains decal objects with tire traces.
+- `Storage_racks` contains storage racks arranged in 2 groups:
+  - `Unpacked` with `Storage_rack` prefabs
+  - `Packs` with `Storage_2x2_Pack` prefabs
+- `Rack_protectors` contains rack protection prefabs
+
+These objects are followed by instances warehouse structural assets:
```

Comment:
```suggestion
These objects are followed by instances of warehouse structural assets:
```

Reply:
Please run grammar / spellcheck for the entire thing to catch some minor issues like this.